src/config.ts

```diff
@@ -103,21 +103,45 @@ export const electronStore = new Store<IElectronStore>({
 });
 
 const LocalServerUrl = (): string => {
-  return `localhost:${LocalServerPort()}`;
+  return `${LocalServerHost().host}:${LocalServerPort().port}`;
 };
 
 const GraphQLServer = (): string => {
   return `${LocalServerUrl}/graphql`;
 };
 
-const RpcServerPort = (): number => {
+const RpcServerHost = (): { host: string; notDefault: boolean } => {
+  const host = process.env.NC_RPC_SERVER_HOST;
+  return host == null
+    ? { host: "127.0.0.1", notDefault: false }
+    : { host, notDefault: true };
+};
+
+const RpcServerPort = (): { port: number; notDefault: boolean } => {
+  const port = process.env.NC_RPC_SERVER_PORT;
+  if (port != null && port.match(/^\d+$/)) {
+    return { port: +port, notDefault: true };
+  }
+
   // FIXME: 열려 있지 않는 랜덤한 포트를 반환하게 해야 합니다.
-  return 23142;
+  return { port: 23142, notDefault: false };
 };
 
-const LocalServerPort = (): number => {
+const LocalServerHost = (): { host: string; notDefault: boolean } => {
+  const host = process.env.NC_GRAPHQL_SERVER_HOST;
+  return host == null
```

Comment:
This comparison will be `true` when also `host` is undefined. Is it intended? If not, how about using `===` and `!==` operator.

Reply:
That's intended. 😎